src/frontend.cpp

```diff
@@ -19,21 +27,29 @@ void Frontend::Exception(Error errorCode, const std::string &input) {
 		Frontend::PrintHelper();
 		break;
 	case fileError:
-		std::cout << "Error: No file for parsing or file not found: " << input
-							<< std::endl;
-		break;
+		if (input.find('-') == std::string::npos) {
```

Comment:
You could use `input[0] == '-'` instead, so file names with dashes in the middle didn't produce problems.